app/containers/Admin/ReportingYear/ReportingYearTable.tsx

```diff
@@ -75,7 +117,7 @@ export const ReportingYearTableComponent: React.FunctionComponent<Props> = (
     <>
       <div style={{textAlign: 'right'}}>
         <Button
-          style={{marginTop: '-220px'}}
+          style={{marginTop: '-330px'}}
```

Comment:
@pbastia Unfortunately I realized too late that we'd have to do some gymnastics to try to use your new [`titleControls` button](https://github.com/bcgov/cas-ciip-portal/pull/1290/files#diff-f6385c6dabb76b0f2eb0463c90bf2433c347a773f6167522cf6ebf99ae1e8e74R108) in DefaultLayout with the reporting years admin page - the action needed for the button is known only within the scope of `ReportingYearTable` and isn't available at the top page level... 🤔

Reply:
What I've done in the meantime was to tweak the currently implemented absolute positioning of the New Reporting Year button slightly to center it vertically between the title header and the table - there was a problem with using that method to put it in its original place (making it impossible to click on the button).

We need to find a better solution in the component hierarchy for these buttons that do operations on an inner page component (adding items to the SearchTable).